src/ytmodal/ytmodalcomponent.js

```diff
@@ -233,4 +241,4 @@ export class YTModalPlayer {
 }
 
 
-export default YTModalComponent;
+export default YTModalComponent;
```

Comment:
might want to update your editor settings so final newline isn't stripped.

Reply:
Fixed